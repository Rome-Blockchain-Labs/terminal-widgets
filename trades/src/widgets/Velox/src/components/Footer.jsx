import React from 'react';

export const Footer = (props) => {
  return (
    <footer
      className="footer"
      style={{
        paddingBottom: '30px',
        paddingTop: '120px',
        textAlign: 'center',
      }}
    >
      &copy; 2022 RBL Velox All Right Reserved
      {/* <div className="footer__inner flex --align-center --just-center">
                <a href="1"
                    className="footer__link --svg__tg">Telegram</a>
                <a href="2"
                    className="footer__link --svg__m">M...</a>
                <a href="3"
                    className="footer__link --svg__twitter">Twitter</a>
            </div> */}
    </footer>
  );
};
