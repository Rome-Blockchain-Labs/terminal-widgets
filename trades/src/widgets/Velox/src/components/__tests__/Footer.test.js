import { shallow } from 'enzyme';

import { Footer } from '../Footer';

describe('Footer component render', () => {
  it('Footer snapshot added', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
