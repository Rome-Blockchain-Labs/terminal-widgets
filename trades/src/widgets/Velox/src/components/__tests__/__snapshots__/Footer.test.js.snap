// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render Footer snapshot added 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__inner flex --align-center --just-center"
  >
    <a
      className="footer__link --svg__tg"
      href="1"
    >
      Telegram
    </a>
    <a
      className="footer__link --svg__m"
      href="2"
    >
      M...
    </a>
    <a
      className="footer__link --svg__twitter"
      href="3"
    >
      Twitter
    </a>
  </div>
</footer>
`;
